dashboard/test/models/pd/workshop_test.rb

```diff
@@ -372,30 +372,7 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     workshop = create(:workshop, :ended)
 
     create(:pd_workshop_participant, workshop: workshop, enrolled: true)
-    Pd::Enrollment.any_instance.expects(:send_exit_survey).never
-
-    workshop.send_exit_surveys
-  end
-
-  test 'send_exit_surveys with attendance but no account gets email for counselor or admin' do
```

Comment:
I removed post-workshop email tests for counselor, admin, admin_counselor, and facilitator workshops as those workshop types are deprecated and we no longer have any ongoing workshops of those types.